src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -926,6 +925,11 @@ private void OnServerTimeout()
 
         private async Task PingServer()
         {
+            if (Debugger.IsAttached)
```

Comment:
Hrm, why are we not sending the Pings while debugging? I get why we don't want to time-out when debugging but I think we should still be sending the pings.